ps4-ksdk/source/kfirmware.c

```diff
@@ -19,31 +19,151 @@ unsigned short kget_firmware_from_base(uint64_t kernbase) {
     char *firmwareString = "firmware";
     
     char *fw0505 = (char *)(kernbase + 0x7C7350);
-	char *fw0672 = (char *)(kernbase + 0x827FB9);
+    char *fw0672 = (char *)(kernbase + 0x827FB9);
+    char *fw0700 = (char *)(kernbase + 0x827145);
+    char *fw0701 = (char *)(kernbase + 0x827145);
     char *fw0702 = (char *)(kernbase + 0x827145);
+    char *fw0750 = (char *)(kernbase + 0x7B5F4C);
+    char *fw0751 = (char *)(kernbase + 0x7B5F4C);
+    char *fw0755 = (char *)(kernbase + 0x7B5F4C);
+    char *fw0800 = (char *)(kernbase + 0x7A0C60);
```

Comment:
Multiple firmwares are using the same address here. We need to either find a unique string or something else to compare against.